stable/redis-ha/templates/redis-haproxy-servicemonitor.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if and ( .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" ) ( .Values.haproxy.metrics.serviceMonitor.enabled ) ( .Values.haproxy.metrics.enabled ) }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+{{- if .Values.haproxy.metrics.serviceMonitor.labels }}
+  labels:
+{{ toYaml .Values.haproxy.metrics.serviceMonitor.labels | indent 4}}
+{{- end }}
```

Comment:
Here you could do:

```
{{- with .Values.haproxy.metrics.serviceMonitor.labels }}
  labels:
{{ toYaml . | nindent 4}}
{{- end }}
```

Which I think is a lot cleaner and you do not have to repeat `.Values.haproxy.metrics.serviceMonitor.labels` twice

Reply:
Thanks, read about `with`, didn't know about this conditional.
Agree with you.